test/jus.js

```diff
@@ -1,5 +1,6 @@
 /* globals describe, it */
 
+const upath      = require('upath')
```

Comment:
Yes! Because you use 'path' as a variable later on in the tests, ~line 74. Frankly, I'd rather rename *that* variable and make this one `const path = require('upath')`. I was just minimizing the LOC changed.